converter/src/main/java/gov/cms/qpp/conversion/model/error/FormattedErrorCode.java

```diff
@@ -0,0 +1,54 @@
+package gov.cms.qpp.conversion.model.error;
+
+import java.util.Objects;
+
+import com.google.common.base.MoreObjects;
+
+public class FormattedErrorCode implements LocalizedError {
+
+	private final ErrorCode errorCode;
+	private final String message;
+
+	public FormattedErrorCode(ErrorCode errorCode, String message) {
+		this.errorCode = errorCode;
+		this.message = message;
+	}
+
+	@Override
+	public ErrorCode getErrorCode() {
+		return errorCode;
+	}
+
+	@Override
+	public String getMessage() {
+		return message;
+	}
+
+	@Override
+	public String toString() {
+		return MoreObjects.toStringHelper(this)
+				.add("errorCode", errorCode)
+				.add("message", message)
+				.toString();
+	}
+
+	@Override
+	public boolean equals(Object o) {
+		if (o == null) {
+			return false;
+		}
+
+		if (o instanceof FormattedErrorCode) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Compare to "this.getClass()" instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2162)